README.md

```diff
@@ -12,6 +12,18 @@
 [![Chat on Discord](https://img.shields.io/badge/chat-discord-blue?style=flat&logo=discord)](https://discord.gg/h2ur8H6mK6)
 </div>
 
+## Table of Contents
+[🎯 Goal](#-goal)
```

Comment:
missing "-"

Reply:
My bad!! I have updated it please review it.